views/file/importdialog.html

```diff
@@ -1,25 +1,31 @@
 <div class="d-flex flex-column flex-fill">
-  <div ng-form="importForm" class="modal-body">
+  <form class="modal-body" name="importForm" ng-submit="importdialog.accept()">
```

Comment:
Not sure if this will have any side effects, but we typically have switched to using `ng-form` in more places. It allows for Angular to support sub-forms for validation purposes, where for some reason validation tends to break when you nest standard `<form>` elements